### ISSUE TITLE
Can't create a new annotation from reads in WebApolloAlignments2 track

### ISSUE THREAD

**Issue by abretaud** [GMOD/Apollo#2197]
```
biotype not found [undefined] converting to mRNA AnnotTrack.js:1032:25
TypeError: test_feature.get(...) is undefined[En savoir plus] 11.bundle.js line 1001 > eval:264:95
    getPreferredSubFeature webpack:///./plugins/WebApollo/js/JSONUtils.js?:264
    createApolloFeature webpack:///./plugins/WebApollo/js/JSONUtils.js?:332
    process webpack:///./plugins/WebApollo/js/View/Track/AnnotTrack.js?:1047
    createAnnotations webpack:///./plugins/WebApollo/js/View/Track/AnnotTrack.js?:1101
    action webpack:///./plugins/WebApollo/js/View/Track/WebApolloAlignments2.js?:25
    handler webpack:///./src/JBrowse/View/Track/BlockBased.js?:693
    onItemClick webpack:///./node_modules/dijit/_MenuBase.js?:318
    postCreate webpack:///./node_modules/dijit/_MenuBase.js?:101
    selector Dojo
```

**Comment by nathandunn** [comment]
Occurs in 2.3.1 and both Canvas and HTML tracks

**Comment by abretaud** [comment]
Fixed by 2.4.0
